javascript/builtins/intl/DateTimeFormat.json

```diff
@@ -496,7 +496,7 @@
                   "notes": "Before version 13.0.0, only the locale data for <code>en-US</code> is available by default. See <a href='https://developer.mozilla.org/docs/Web/JavaScript/Reference/Global_Objects/Intl/DateTimeFormat/DateTimeFormat'>the <code>DateTimeFormat()</code> constructor</a> for more details."
                 },
                 "opera": {
-                  "version_added": false
+                  "version_added": "63"
```

Comment:
FYI, I also tested this on Opera current version and it also works. Makes no sense to me this would work on opera android but not opera, and would not match Chrome. So I have marked this as going in the same/appropriate Blink version.